src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml

```diff
@@ -218,6 +218,33 @@ else
         </div>
     </fieldset>
 </div>
+
+<div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.EmploymentEndDate, "govuk-form-group--error")">
+    <fieldset class="govuk-fieldset" aria-describedby="finish-date-hint" role="group">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+            Planned end date for this employment
+        </legend>
+        <span id="employment-finish-date-hint" class="govuk-hint">
+            For example, 09 2019
+        </span>
+        @Html.ValidationMessageFor(m => m.EmploymentEndDate, null, new { @class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.EmploymentEndDate) })
+        <div class="govuk-date-input">
+            <div class="govuk-date-input__item">
+                <div class="govuk-form-group">
+                    <label asp-for="EmploymentEndMonth" class="govuk-label govuk-date-input__label">Month</label>
+                    <input asp-for="EmploymentEndMonth" min="1" max="12" class="govuk-input govuk-date-input__input govuk-input--width-2" type="number">
+                </div>
+            </div>
+            <div class="govuk-date-input__item">
+                <div class="govuk-form-group">
+                    <label asp-for="EmploymentEndYear" class="govuk-label govuk-date-input__label">Year</label>
+                    <input asp-for="EmploymentEndYear" min="1900" max="9999" class="govuk-input govuk-date-input__input govuk-input--width-4" type="number">
+                </div>
+            </div>
+        </div>
+    </fieldset>
+</div>
+
```

Comment:
I do apologise I haven't ran this, but is this just appearing for the Flexible, it shouldn't appear on Regular?